app/models/serializers/work_queue/appeal_serializer.rb

```diff
@@ -80,6 +80,16 @@ class WorkQueue::AppealSerializer
 
   attribute :cavc_remand
 
+  attribute :remand_source_appeal_id do |appeal|
+    if !appeal.cavc_remand.nil?
```

Comment:
nitpick - take it or leave it!
```suggestion
    if appeal.cavc_remand.present?
```

Reply:
Thanks! I always forget about those ruby shortcuts